jalm00-sp2

MJalovec opened the pull request:
Link to application: https://eso.vse.cz/~jalm00/sp2

MJalovec committed:
jalm00-sp2

Link to application: https://eso.vse.cz/~jalm00/sp2

nvbach91 commented:
- mohl byste nadesignovat aplikace trochu lepe aby uzivatel nemusel scrollovat dolu po kazde zmene filtru?
- zpusob vyberu ingredience je velice neprehledne, pouzivejte misto checkboxu napr multi-select nebo select s vyhledavacim polem, melo by to fungovat tak, ze kdyz se zvoli ingredience tak se zobrazi v nejakem kontejneru ze ktereho se da odebrat
- detaily o jednom drinky zobrazujte jako modalove okno, nebo jako postranni lista
- pevne ukotvene elementy by se nemely hybat s kazdym klikem na filter...
- odstrante nebo zakomentujte veskere console.log() z kodu
- pouzivejte useQuery z tanstacku pro stahovani dat z API
- osetrete neexistujici ingredience v jednom drinku
- BASE_URL ukladejte do konstanty

MJalovec committed:
version 02

-- Přidaný loading spinner při všech API calls.
-- Přidané modalové okno pro drink
-- Zlepšen UI/UX design drinku
-- Přidaný menubar s vybranými ingrediencemi, který zlepšuje zážitek uživatele zpřijemňujícím prvkem lehkého odebrání filtrované ingredience
-- Opraven bug v tabulce s ingrediencemi, kde při psaní do inputu se zmenšoval a zvětšoval height pole z tohoto důvodu byl zážitek uživatele chaotický
-- Odstraněny / Zakomentovány všechny console.logy.
-- Přidaná konstantní proměnná s BASE_URL pro veškeré API cally.


Děkuji za možnost opravy. 

S pozdravem 
MJ.
